README/MATH/Prediction_Market_Math.tex

```diff
@@ -0,0 +1,1379 @@
+\documentclass{article}
+\usepackage{tikz}
+\usetikzlibrary{calc}
+\usetikzlibrary{shapes.geometric, arrows, positioning}
+\usepackage{amsmath}
+
+\title{Social Predict - Weighted Probability Adjustment Model (WPAM) for Market Pricing and Divergence-Based Payout Model (DBPM) for Payout Distributions}
+\author{Patrick Delaney}
+\date{November 2024}
+
+% Define the macro to draw pie charts
+\newcommand{\piechart}[3]{ % #1: x-coordinate, #2: y-coordinate, #3: fraction (0 to 1)
+  \draw (#1,#2) circle (0.5); % Draw circle at (#1,#2) with radius 0.5
+  \pgfmathsetmacro{\startangle}{90}
+  \pgfmathsetmacro{\endangle}{90 - 360 * #3} % Calculate end angle based on fraction
+  \filldraw[fill=cyan!50] (#1,#2) -- ++(\startangle:0.5) arc (\startangle:\endangle:0.5) -- cycle; % Draw filled sector
+}
+
+\begin{document}
+
+\maketitle
+
+\section{Overview}
+
```

Comment:
The 3 items are good. I think Overall Purpose and Transparency and Unit Testing can be made more concise - e.g.

This document describes how market pricing (probability) gets set and how points get distributed from a series of historical bets. It aims to be clear and provide graphical illustrations.

Pricing and distribution underpin this software. Making the full algorithm understandable is essential to writing proper unit tests and clear documentation.

Just in general, short sentences + active voice